frontend/javascripts/oxalis/model/sagas/quick_select_saga.ts

```diff
@@ -32,13 +32,13 @@ export default function* listenToQuickSelect(): Saga<void> {
     "COMPUTE_QUICK_SELECT_FOR_RECT",
     function* guard(action: ComputeQuickSelectForRectAction) {
       try {
-        // As changes to the volume layer will be applied, the potentially existing mapping should be pinned to ensure a consistent state.
         const volumeTracing: VolumeTracing | null | undefined = yield* select(
           getActiveSegmentationTracing,
         );
         if (volumeTracing) {
+          // As changes to the volume layer will be applied, the potentially existing mapping should be locked to ensure a consistent state.
           const { isMappingPinnedIfNeeded } = yield* call(
```

Comment:
It looks like the search and replace was not exhaustive, could you check again for remaining occurrences of "pinned"? You could also use the "Files Changed" tab here on github.